src/main/java/seedu/mtracker/MTracker.java

```diff
@@ -65,4 +65,5 @@ public void executeProgram() {
     public static void main(String[] args) {
         new MTracker().executeProgram();
     }
+    //@@author
```

Comment:
What is this ending author tag for? If I am not wrong it either checks the next author or end of file whichever comes first.  this is at end of file alr